advocacy_docs/partner_docs/VisionXGuide/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+VisionX is the most flexible and independent low-code platform available, enabling both business users and professional developers to visually develop web, desktop and native mobile applications extremely fast. These can be very simple applications that replace paper processes or Excel sheets, easy-to-use forms on ERP systems, dashboards, mobile apps, and even  highly complex billing applications, customer portals or trading systems. 
+
+The VisionX low-code platform supports major databases, such as Postgres, EDB, Oracle, MS SQL, MySQL and SAP HANA. VisionX is automatically bundled with a Postgres database and is a low-code platform for Postgres.
```

Comment:
change to
The VisionX low-code platform supports major databases, such as Postgres, EDB Postgres Advanced Server and EDB Postgres Extended Server. VisionX is automatically bundled with a Postgres database and is a low-code platform for Postgres.